Amplify/DevMenu/View/DeviceInfoRow.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SwiftUI
+
+@available(iOS 13.0.0, *)
+struct DeviceInfoRow: View {
+    var rowItem: DeviceInfoItem
+
+    var body: some View {
+        VStack {
+            HStack {
+                Text(rowItem.key).bold()
+                Spacer()
+            }
+            HStack {
+                Text(rowItem.value)
+                Spacer()
+            }
+        }
+    }
```

Comment:
it seems like this can be simplified:

```swift
VStack {
    Text(rowItem.key).bold()
    Spacer()
    Text(rowItem.value)
}.padding()
```

Or something like that :)